crates/turbopack-tests/tests/snapshot/cssmodules/composes/output/79fb1_turbopack-tests_tests_snapshot_cssmodules_composes_input_index_module_7d7e1c.css

```diff
@@ -1,9 +1,3 @@
 /* [project]/crates/turbopack-tests/tests/snapshot/cssmodules/composes/input/index.module.css [test] (css) */
-.className__index__286d8691 {
-  background: red;
-  color: yellow;
-}
-.subClass__index__286d8691 {
-  background: blue;
-}
+.index-module__Tw9iRq__className{background:red;color:#ff0}.index-module__Tw9iRq__subClass{background:#00f;}
```

Comment:
I would prefer the formatting where `className` is in the front. That's much cleaner. Could we configure that for lightningcss?